src/vscode-dts/vscode.d.ts

```diff
@@ -19086,6 +19086,12 @@ declare module 'vscode' {
 		 * string-manipulation of the prompt.
 		 */
 		readonly references: readonly ChatPromptReference[];
+
+		/**
+		 * This is the model that is currently selected in the UI. Extensions can use this or use {@link chat.selectChatModels} to
+		 * pick another model. Don't hold onto this past the lifetime of the request.
+		 */
+		readonly model: LanguageModelChat;
```

Comment:
I made it non-optional. Do we keep this, or make it undefined for inline chat and other surfaces? Those should get the model picker at some point though.